src/coreclr/vm/riscv64/stubs.cpp

```diff
@@ -1932,7 +1932,7 @@ PCODE DynamicHelpers::CreateDictionaryLookupHelper(LoaderAllocator * pAllocator,
             p += 4;
         }
 
-        BYTE* pBLTCall = NULL;
+        BYTE* pBLECall = NULL;
```

Comment:
Right. I thought what is correct.
Why I choose `pBLECall` are
- In other archs, it uses `pBLECall` instead of `pBGECall` even if the instruction is `bge` like LOONGARCH.
- I think what this code really want to say is `jump if size <= offset` (original codes in ARM64). However there is no instruction for 'BLE' in RISC-V, so it swaps `size` and `offset` and change `ble` to `bge`.
So I just keep this to `pBLECall`. If you still think `pBGECall` is better, I will update.

Reply:
Ah, got it. If this approach is already used in other arches, I thinks it's fine to keep it. Or all arches can be updated. Both approaches are fine to me, so up to you.